src/main/java/core/basesyntax/FruitTransaction.java

```diff
@@ -0,0 +1,46 @@
+public class FruitTransaction {
```

Comment:
The class name `FruitTransaction` is quite generic. Consider renaming it to reflect the specific type of transactions it's meant to represent, such as `FruitOperationRecord` or `FruitTransactionEntry`, to give more context.